src/Microsoft.AspNet.Razor/Parser/SyntaxTree/Block.cs

```diff
@@ -19,7 +19,7 @@ public Block(BlockBuilder source)
             source.Reset();
         }
 
-        protected Block(BlockType? type, IEnumerable<SyntaxTreeNode> contents, IParentChunkGenerator generator)
+        protected Block(BlockType? type, List<SyntaxTreeNode> contents, IParentChunkGenerator generator)
```

Comment:
Do we want to make these consistent and use `IReadOnlyList<>` here? (like the other constructor)